package/kedro_viz/launchers/cli/main.py

```diff
@@ -0,0 +1,32 @@
+"""`kedro_viz.launchers.cli.main` is an entry point for Kedro-Viz cli commands."""
+
+import click
+
+from kedro_viz.launchers.cli.lazy_default_group import LazyDefaultGroup
+
+
+@click.group(
+    name="Kedro-Viz",
+    cls=LazyDefaultGroup,
+    lazy_subcommands={
+        "viz": "kedro_viz.launchers.cli.main.viz",
+    },
+)
+def viz_cli():  # pylint: disable=missing-function-docstring
+    pass
+
+
+@viz_cli.group(
+    name="Kedro-Viz",
+    cls=LazyDefaultGroup,
+    lazy_subcommands={
+        "run": "kedro_viz.launchers.cli.run.run",
+        "deploy": "kedro_viz.launchers.cli.deploy.deploy",
+        "build": "kedro_viz.launchers.cli.build.build",
+    },
+    default="run",
+    default_if_no_args=True,
+)
+@click.pass_context
+def viz(ctx):  # pylint: disable=unused-argument
```

Comment:
@ravi-kumar-pilla Q: Why're both the classes lazy?

Reply:
As discussed, I thought there is some benefit having LazyGroup for the entry point. But it does not seem to affect kedro commands or viz commands. Removed the LazyGroup on viz_cli. Thank you for raising this.